priv/styles.css

```diff
@@ -95,13 +95,18 @@ h1, h2, h3, h4, h5, h6 {
   border-radius: 100px 0 0 100px;
 }
 
+@media screen and (min-width: 60em) {
+  .search-form input[type=search] {
+    min-width: 300px;
+  }
+}
+
 .search-form input[type=submit] {
   background: var(--color-white);
   border: none;
   border-radius: 0 100px 100px 0;
   height: 30px;
   padding: 0 var(--gap-s);
-  margin-left: 1px;
```

Comment:
This change was to remove the tiny gap between the search input and the submit button:

<img width="321" alt="Screenshot 2023-05-28 at 1 31 20 PM" src="https://github.com/gleam-lang/packages/assets/1486634/ea95f649-8558-4e5f-8720-3fc568bc0f30">

It was driving me nuts 😅 

I can remove it from this PR if this margin is intentional or my removal of it is controversial.